src/main/java/org/jabref/gui/util/ThemeLoader.java

```diff
@@ -51,6 +55,9 @@ public ThemeLoader(FileUpdateMonitor fileUpdateMonitor) {
      */
     public void installBaseCss(Scene scene, JabRefPreferences preferences) {
         addAndWatchForChanges(scene, DEFAULT_PATH_MAIN_CSS, 0);
+        addAndWatchForChanges(scene,DARK_THEME_PATH_CSS,0);
```

Comment:
It should work without copying (and modifying) the css files if you add here something like:
````
if (preferences.useDarkTheme) {
   addAndWatchForChanges(scene,DARK_THEME_PATH_CSS, 1);
}
````

Reply:
Do we really need to watch for changes? Maybe just have a second method which just adds/loads the css and doesn't  watch for changes.

Reply:
It only watches for changes if the corresponding command line flag is passed, i.e. only for us developers.